src/examples.js

```diff
@@ -0,0 +1,73 @@
+export const BLANK = "Pick an example...";
+export const INTRO = "Intro";
+export const EXAMPLES = {
+  [BLANK]: "",
+
+  [INTRO]: `// Type your Decimal-using code here!
+// It will execute, and any console output
+// will appear on the right.
+
+// To see the transpiled code, click
+// "Toggle Output" above.
+
+// You can also manipulate the DOM from
+// this code, and see the results with
+// "Toggle DOM".
+
+// If you just want to see some code that
+// uses Decimals, try picking one of the
+// examples from the dropdown menu at the
+// top right.
+
+function snowCactusEscapade(a) {
+    return 11.1m + a;
+}
+
+let x = snowCactusEscapade(12m);
+console.log(x);
+console.log(x.toString());
+`,
+
+  "Scientific notation": String.raw`// Format a number using scientific notation
+function scino(num, maximumFractionDigits) {
+  num = Decimal(num);
```

Comment:
I don't think I normally have this rule enabled in production ... happy to switch it on, but this is why I think we should pick a linter configuration and then just never think about it anymore.

(It wouldn't have caught it because of keeping the code in a string, though. I wonder if we can lint the example code strings programmatically as part of the tests — I'll try to figure out how to do that without having eslint choke on the Decimal literals.)